CHANGELOG.md

```diff
@@ -9,9 +9,9 @@ _**Important:** This file contains a draft to the official release notes, and is
 not a stable documentation of the upcoming version (as it will frequently change during
 development)_
 
-## [Unreleased]
+## [2.0.0 - <16.07.2021>]
```

Comment:
The date is outdated now, unfortunately.

Also, I think there should still be an “unreleased” section, and the “expected next version” doesn’t belong below the 2.0.0 heading in my opinion – I would prefer adding the 2.0.0 heading between that line and the “added” heading. (But maybe Tonina disagrees?)

Reply:
Yep, I tried to release it last week but it was kinda late

Reply:
@lucaswerkmeister ok I've done it, please review one more time